src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -402,7 +453,9 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
               <RawText>{asset.name}</RawText>
             </Stack>
             <Row.Value>
-              <Amount.Crypto value={state.deposit.cryptoAmount} symbol={asset.symbol} />
+              <Skeleton isLoaded={!!amountCryptoHumanMinusFees}>
+                <Amount.Crypto value={amountCryptoHumanMinusFees} symbol={asset.symbol} />
```

Comment:
Do we want to display this amount fee-deducted? I felt like yes so I did it but wdyt?